invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/EditButton.js

```diff
@@ -28,7 +28,13 @@ export const EditButton = (props) => {
   };
 
   return (
-    <Button color="orange" size="mini" onClick={handleClick} loading={loading}>
+    <Button
+      fluid
+      color="orange"
```

Comment:
do we have any primary/secondary color set for edit in the theme instead of hardcoding ?